todo-mvc/src/widgets/createTodoItem.ts

```diff
@@ -61,7 +65,14 @@ const createTodoItem = createWidgetBase
 							state: { classes: [ 'toggle' ], checked }
 						}),
 						d(createLabel, {
-							listeners: { dblclick: () => { todoEdit.do(state); } },
+							listeners: {
+								dblclick: () => { todoEdit.do(state); },
+								keypress: (event: KeyboardEvent) => {
+									if (event.which === 13) {
```

Comment:
Yes, but KeyboardEvent.key (the recommended one) isn't supported in Safari or the Android Browser yet.